packages/dds/merge-tree/src/client.ts

```diff
@@ -722,10 +731,10 @@ export class Client {
         resetOp: ops.IMergeTreeDeltaOp,
         segmentGroup: SegmentGroup): ops.IMergeTreeDeltaOp[] {
         assert(!!segmentGroup, "Segment group undefined");
-        const NACKedSegmentGroup = this.mergeTree.pendingSegments.dequeue();
+        const NACKedSegmentGroup = this.mergeTree.pendingSegments!.dequeue();
```

Comment:
```suggestion
        const NACKedSegmentGroup = this.mergeTree.pendingSegments?.dequeue();
```

Reply:
rather NACKedSegmentGroup be undefined, and fail the below assert, rather than null ref exception here